ph5/utilities/tests/test_segd2ph5.py

```diff
@@ -6,28 +6,20 @@
 import sys
 import shutil
 import tempfile
-import logging
-from StringIO import StringIO as StringBuffer
 from mock import patch
-from ph5 import logger, ch as CH
 from ph5.utilities import segd2ph5, tabletokef
 from ph5.core import experiment, segdreader
+from ph5.core.tests.test_base import PH5TestCase
 
 
-class TestSegDtoPH5(unittest.TestCase):
+class TestSegDtoPH5(PH5TestCase):
     def initialize_ph5(self, editmode):
         EX = experiment.ExperimentGroup(nickname="master.ph5")
         EX.ph5open(editmode)
         EX.initgroup()
         return EX
 
     def setUp(self):
-        # capture log string into log_capture_string
-        self.log_capture_string = StringBuffer()
-        logger.removeHandler(CH)
-        ch = logging.StreamHandler(self.log_capture_string)
-        logger.addHandler(ch)
-
         # create tmpdir
```

Comment:
I am thinking about applying tempdir stuff independently with logging handler stuff. Because sometimes we don't need to use both. May be just a function to call.